public/index.html

```diff
@@ -27,249 +27,264 @@
+            <div class="image">
+              <img src="./img/icon-skills.png" alt="Skills">
+            </div>
+            <div class="details">
+              <h2>Skills</h2>
+              <ul class="items">
+                <li>Codekit</li>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>GitLab</li>
+              </ul>
+            </div>
           </div>
-          <ul class="items">
-            <li>Codekit</li>
-            <li>GitHub</li>
-            <li>CodePen</li>
-            <li>GitLab</li>
-            <li>GitLab</li>
-          </ul>
+
         </div>
       </div>
     </section>
-    <section class="about-wraper contact">
-      <h1>
-        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-      </h1>
-      <form action="https://formspree.io/f/xpzkrabn" method="POST">
-        <div class="contact-inputs">
-          <input type="text" name="fullname" maxlength="30" required placeholder="Full name">
-          <input type="email" name="email" minlength="5" required placeholder="Email address">
-          <textarea maxlength="500" name="message" placeholder="Enter text here"></textarea>
+    <section id="contact">
+      <div class="contact-wrapper">
+        <div class="details">
+          <h6>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </h6>
         </div>
-        <div class="project-button">
-          <button type="submit" id="submit-button">Get in Touch</button>
-        </div>
-      </form>
-      <div class="social-icons">
-        <!-- Social networking links-->
-        <ul class="social end-icons">
-          <li>
-            <a href="#"><i class="fab fa-twitter" aria-hidden="true"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fab fa-medium" aria-hidden="true"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fab fa-github" aria-hidden="true"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fab fa-angellist" aria-hidden="true"></i></a>
-          </li>
-        </ul>
+        <form action="https://formspree.io/f/xpzkrabn" method="POST">
+          <div class="user-names">
+            <input type="text" name="firstname" maxlength="30" required placeholder="Fisrt name">
+            <input type="text" name="lastname" maxlength="30" required placeholder="Last name">
+          </div>
+          <div class="input-wrapper">
+            <input type="email" name="email" minlength="5" required placeholder="Email address">
+          </div>
+          <div class="input-wrapper">
+            <textarea maxlength="500" name="message" placeholder="Enter text here"></textarea>
+          </div>
```

Comment:
Please make sure the form matches the given design as shown below
### Original design
![right](https://user-images.githubusercontent.com/58376020/121723761-37f52180-cae7-11eb-8759-51148a270e5b.png)

#### Your project
![wrong](https://user-images.githubusercontent.com/58376020/121723802-46433d80-cae7-11eb-952a-4ccfc7f0a547.png)